base_local_planner/src/obstacle_cost_function.cpp

```diff
@@ -136,9 +137,11 @@ double ObstacleCostFunction::footprintCost (
     return -7.0;
   }
 
-  double occ_cost = std::max(std::max(0.0, footprint_cost), double(costmap->getCost(cell_x, cell_y)));
+  // double occ_cost = std::max(std::max(0.0, footprint_cost), double(costmap->getCost(cell_x, cell_y)));
```

Comment:
Do we want to use the old value normalized or does the max footprint/cost cause other issues?